triss-core/src/main/kotlin/com/pp/trisscore/service/ComparisonService.kt

```diff
@@ -12,8 +12,19 @@ import org.springframework.stereotype.Service
 @Service
 class ComparisonService {
 
-    fun compareApplicationsInfo(dbApplicationInfo: ApplicationInfo, reqApplicationInfo: ApplicationInfo, role: Role) {
-        compareApplications(dbApplicationInfo.application, reqApplicationInfo.application, role)
+    fun compareApproveApplicationsInfo(dbApplicationInfo: ApplicationInfo, reqApplicationInfo: ApplicationInfo, role: Role) {
+        compareApplications(dbApplicationInfo.application, reqApplicationInfo.application,role)
+        compareApproveApplication(role, dbApplicationInfo.application, reqApplicationInfo.application)
```

Comment:
If second method names "Rejected" ( past time) it would be nice if you could name it "compareApprovedApplication"